sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -222,6 +261,8 @@ auto get_native(const SyclObjectT &Obj)
 It is currently supported for SYCL ```platform```, ```device```, ```context```, ```queue```, ```event```,
 ```kernel_bundle```, and ```kernel``` classes. 
 
+The ```get_native(queue)``` function returns either ```ze_command_queue_handle_t``` or ```ze_command_list_handle_t```, depending on whether the SYCL queue uses a Level Zero immediate command list. This depends on whether the environment variable ```SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS``` is set and on whether the queue was created with ```make_queue()```.

```

Comment:
It is not clear how exactly it depends? And what is the SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS default?